src/DeviceBindings.cpp

```diff
@@ -20,12 +20,10 @@ void DeviceBindings::bind(pybind11::module& m){
         .def(py::init<const Pipeline&, const std::string&>(), py::arg("pipeline"), py::arg("pathToCmd"))
         .def(py::init<const Pipeline&, const DeviceInfo&, bool>(), py::arg("pipeline"), py::arg("deviceDesc"), py::arg("usb2Mode") = false)
         .def(py::init<const Pipeline&, const DeviceInfo&, std::string>(), py::arg("pipeline"), py::arg("deviceDesc"), py::arg("pathToCmd"))
-        .def(py::init<const Pipeline&, const DeviceInfo&, bool>(), py::arg("pipeline"), py::arg("deviceDesc"), py::arg("usb2Mode") = false)
-        .def(py::init<const Pipeline&, const DeviceInfo&, std::string>(), py::arg("pipeline"), py::arg("deviceDesc"), py::arg("pathToCmd"))
         .def("isPipelineRunning", &Device::isPipelineRunning)
         .def("startPipeline", &Device::startPipeline)
-        .def("getOutputQueue", &Device::getOutputQueue, py::arg("name"), py::arg("maxSize") = 120, py::arg("overwrite") = false )
-        .def("getInputQueue", &Device::getInputQueue,  py::arg("name"), py::arg("maxSize") = 120, py::arg("overwrite") = false )
+        .def("getOutputQueue", &Device::getOutputQueue, py::arg("name"), py::arg("maxSize") = 16, py::arg("blocking") = true )
```

Comment:
Just to not forget, will need to update the examples for the `overwrite` -> `blocking` change.
https://github.com/luxonis/depthai-experiments/blob/67b7e03/gen2_examples/01_rgb_preview.py#L27
`q_rgb = device.getOutputQueue(name="rgb", maxSize=4, overwrite=True)`
CC @VanDavv